static/js/common/upload-addon.js

```diff
@@ -382,7 +382,7 @@
                                 $upload_field.trigger("upload_finished", [file]);
                             }
                         });
-                    }, 1000);
+                    }, 5000);
```

Comment:
I increased the timeout to reduce the chance we're reading while the server is writing but I don't see any evidence that this is an issue.